true-index.html

```diff
@@ -7,7 +7,10 @@
 <div class="nav-hero-container">
     <div class="hero">
         <div class="container">
+            <h1 class="hero-title">Istio</h1>
+            <img class="hero-logo" src="{{home}}/img/istio-logo.svg" />
```

Comment:
yes it does for assistive/screen reader, I fixed it on master

I asked @sebastienvas it does seem to be depending on CLA check somehow